serving/server.py

```diff
@@ -0,0 +1,46 @@
+import copy
+
+import joblib
+from backend.calculate_level import calculate_level
+from fastapi import FastAPI
+from starlette.middleware.cors import CORSMiddleware
+
+
+app = FastAPI()
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+model = joblib.load(filename="../saved_models/current_model.pkl")
+properties = {}
+
+
+@app.get("/", tags=["root"])
+async def read_root() -> dict:
+    return {}
+
+
+@app.get("/properties")
+async def get_properties() -> dict:
+    return properties
```

Comment:
Polecam zrobić `api_models.py` czy coś w tym stylu i tam zrobić typy Pydantica, FastAPI ma go wbudowanego. Wtedy od razu macie walidację typów przyjmowanych i zwracanych. Działa to bardzo podobnie do TypedDict albo dataclass